public/app/features/dashboard-scene/scene/VariableControls.tsx

```diff
@@ -60,10 +60,16 @@ export function VariableValueSelectWrapper({ variable, inMenu }: VariableSelectP
     }
 
     // Ignore click if it's inside the value control
-    if (evt.target instanceof Element && !evt.target.closest(`label`)) {
-      // Prevent clearing selection when clicking inside value
-      evt.stopPropagation();
-      return;
+    if (evt.target instanceof Element) {
+      // multi variable options contain label element so we need a more specific
+      //  condition to target variable label to prevent edit pane selection on option click
+      const testId = evt.target.closest('label[data-testid]')?.getAttribute('data-testid');
+
+      if (!testId?.startsWith(selectors.pages.Dashboard.SubMenu.submenuItemLabels(''))) {
+        // Prevent clearing selection when clicking inside value
+        evt.stopPropagation();
+        return;
+      }
```

Comment:
I don't like this use of testId. In my opinion testId should really only be used in testing.

I'm thinking we could remove this check completely and surround the component with an element that stops propagation.

something like this (but not with the inline function)
<img width="971" height="891" alt="image" src="https://github.com/user-attachments/assets/de5caba5-f1cb-4c7f-bb54-4e66b3638138" />

Reply:
Good catch on the border issue, I didn't pick that up. Let's go with option 3 to keep the E2E tests the same.